### PR TITLE
nimbus book: add page on setting up a systemd service (WIP)

### PR DESCRIPTION
**Pull request by sachayves**
builds off @zah 's wiki entry

### CODE REVIEW

**Comment by stefantalpalaru** [comment]
It should probably mention that eth2stats is not required for beacon_node to function, and how to register your node on https://eth2stats.io/add-node

Also, shame we can't get a SIGINT to the beacon_node process to allow it to shut down gracefully.

**Comment by sachayves** [comment]
added a prerequisites section and a step on registering your node
(i don't think we need to explicitly mention that the beacon node can run without eth2stats)

final review would be appreciated

File: docs/the_nimbus_book/src/beacon_node_systemd.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+```
+
+The resulting binaries should appear in `nim-beacon-chain/build/beacon_node` and `eth2stats-client/eth2stats-client`, respectively.
+
+### 3. Register your node
+
+Add your node to eth2stats and run a data collector app that connects to your beacon chain client.
+
+```
+./eth2stats-client run \
```
Comment (stefantalpalaru) [inline docs/the_nimbus_book/src/beacon_node_systemd.md:43]: This doesn't look right. We already run `eth2stats-client` inside the new shell script.

**Comment by stefantalpalaru** [comment]
> The bump of eth2-testnets is confusing me, but as far as I can tell, it shouldn't be there.

Maybe it's a lack of rebase on devel, but I wouldn't trust the UI to merge this PR. Better do it by hand, rebase and diff it first.

**Comment by sachayves** [comment]
> The bump of eth2-testnets is confusing me, but as far as I can tell, it shouldn't be there.

@zah , @stefantalpalaru i think the changes in `vendor` happen when i run `merge/upstream devel` from my local devel branch (with upstream set to `https://github.com/status-im/nim-beacon-chain.git`). confusing me too

**Pull request closed without merging**